oneandone/src/test/java/org/apache/jclouds/oneandone/rest/features/LoadBalancerApiLiveTest.java

```diff
@@ -0,0 +1,211 @@
+              .persistence(Boolean.TRUE)
+              .persistenceTime(200)
+              .build());
+   }
+
+   @Test
+   public void testList() {
+      loadBalancers = loadBalancerApi().list();
+
+      assertNotNull(loadBalancers);
+      assertFalse(loadBalancers.isEmpty());
+      Assert.assertTrue(loadBalancers.size() > 0);
+   }
+
+   public void testListWithOption() {
+      GenericQueryOptions options = new GenericQueryOptions();
+      options.options(0, 0, null, "jclouds", null);
+      List<LoadBalancer> resultWithQuery = loadBalancerApi().list(options);
+
+      assertNotNull(resultWithQuery);
+      assertFalse(resultWithQuery.isEmpty());
+      Assert.assertTrue(resultWithQuery.size() > 0);
+   }
+
+   public void testGet() {
+      LoadBalancer result = loadBalancerApi().get(currentLoadBalancer.id());
+
+      assertNotNull(result);
+      assertEquals(result.id(), currentLoadBalancer.id());
+   }
+
+   @Test(dependsOnMethods = "testGet")
+   public void testUpdate() throws InterruptedException {
+      String updatedName = "UpdatedjcloudsPolicy";
+
+      LoadBalancer updateResult = loadBalancerApi().update(currentLoadBalancer.id(), LoadBalancer.UpdateLoadBalancer.builder()
+              .name(updatedName)
+              .description("description")
+              .healthCheckInterval(40)
+              .healthCheckTest(Types.HealthCheckTestTypes.TCP)
+              .method(Types.LoadBalancerMethod.ROUND_ROBIN)
+              .persistence(Boolean.TRUE)
+              .persistenceTime(1200)
+              .build());
+
+      assertNotNull(updateResult);
+      assertEquals(updateResult.name(), updatedName);
+
+   }
+
+   @Test(dependsOnMethods = "testUpdate")
+   public void testAssignServerIp() throws InterruptedException {
+      assertNodeAvailable(currentServer);
+      currentServer = updateServerStatus(currentServer);
+      List<String> ips = new ArrayList<String>();
+      ips.add(currentServer.ips().get(0).id());
+      LoadBalancer.ServerIp.CreateServerIp toAdd = LoadBalancer.ServerIp.CreateServerIp.create(ips);
+
+      LoadBalancer updateResult = loadBalancerApi().assignServerIp(currentLoadBalancer.id(), toAdd);
+      assertEquals(updateResult.serverIps().get(0).id(), currentServer.ips().get(0).id());
+      assertNotNull(updateResult);
+
+   }
+
+   @Test(dependsOnMethods = "testAssignServerIp")
+   public void testListServerIps() {
+      List<LoadBalancer.ServerIp> servers = loadBalancerApi().listServerIps(currentLoadBalancer.id());
+
+      assertNotNull(servers);
+      assertFalse(servers.isEmpty());
+      Assert.assertTrue(servers.size() > 0);
+   }
+
+   @Test(dependsOnMethods = "testAssignServerIp")
+   public void testServerIpGet() {
+      LoadBalancer.ServerIp result = loadBalancerApi().getServerIp(currentLoadBalancer.id(), currentServer.ips().get(0).id());
+
+      assertNotNull(result);
+   }
+
+   @Test(dependsOnMethods = "testServerIpGet")
```

Comment:
This should depend on the list and get tests, to prevent race conditions in the list one.